persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -68,6 +68,7 @@ CREATE TABLE MessageMention (
       PRIMARY KEY (message_id, conversation_id, start) -- there should not be any overlapping mentions
 );
 CREATE INDEX message_mentioned_user_id_index ON MessageMention(user_id);
+CREATE INDEX message_mentioned_message_and_user_id_index ON MessageMention(message_id, user_id);
```

Comment:
question: do we need the index in this order?
if not we can flip it and remove the other index message_mentioned_user_id_index